jupyter_console/interactiveshell.py

```diff
@@ -574,8 +622,7 @@ def interact(self, display_banner=None):
                 self.showtraceback()
             else:
                 try:
-                    self.input_splitter.push(line)
-                    more = self.input_splitter.push_accepts_more()
+                    more, indent = self.handle_source(line)
```

Comment:
@dsblank we carefully got `indent` back here, but then we haven't actually wired it up to the prompt. Have you got time to have a look? There's a method `_indent_current_str()` on the parent class that I think we'd need to define on this subclass.